Deployment/Kombit.Samples.CH.WebsiteDemo/packages.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Kombit.Samples.BasicPrivilegeProfileParser" version="1.0.0.0" targetFramework="net45" />
+  <package id="Kombit.Samples.Common" version="1.0.0.2" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net45" />
```

Comment:
Please upgrade to .NET 4.8 as well as latest version (5.2.7 or so)

Reply:
Same for other packages

Reply:
upgraded to .net 4.8 and updated packages to newer version